src/components/login-form/login-form.tsx

```diff
@@ -0,0 +1,76 @@
+"use client";
+import {
+  CardTitle,
+  CardDescription,
+  CardHeader,
+  CardContent,
+  CardFooter,
+  Card,
+} from "@/components/ui/card";
+import { Label } from "@/components/ui/label";
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import { useForm, SubmitHandler } from "react-hook-form";
+
+interface IFormInput {
+  email: string;
+  password: string;
+}
```

Comment:
General question - do we use "I" naming convention for interaces or not? imo it is redundant, as types have clear enough distinction from variables

Reply:
Thats true, i am used to naming like "FormInput" so we can stick to that

Reply:
Done